plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1318,15 +1327,19 @@ void chain_plugin::plugin_startup()
    try {
       auto shutdown = [](){ return app().quit(); };
       auto check_shutdown = [](){ return app().is_quiting(); };
-      if (my->snapshot_path) {
+      auto bvc_plug = app().find_plugin<blockvault_client_plugin>();
+      EOS_ASSERT( bvc_plug, plugin_config_exception, "blockvault_client_plugin not found" );
```

Comment:
fwiw, I think you can `get_plugin("blockvault_client_plugin")` which throws if plugin not found. will save a line or two

Reply:
blockvault_client_plugin is no longer required, so this is no longer relevant